### PR TITLE
closes #65: change "nameless" to <bot owner nick>

### LINKED ISSUE #65: add option to config so "nameless's trivia bot" string will show admin name
If you'd like this feature added, I'll take care of it.

### PR DESCRIPTION
**Pull request by andy5995**
Also fixes this message when admin uses ?help "_cmsg() takes exactly 3
arguments (2 given)" (L240)

### CODE REVIEW

File: trivia.py [hunk 1]
```diff
@@ -229,14 +230,14 @@ def _help(self, args, user, channel):
         try:
             self._admins.index(user)
         except:
-            self._cmsg(user, "I'm nameless's trivia bot.")
+            self._cmsg(user, "I'm %s's trivia bot." % config.OWNER)
```
Comment (rawsonj) [inline trivia.py]: Hi Andy,

Could you use `I'm {}'s trivia bot.".format(config.OWNER)` instead? The % syntax is old and the format() method is preferred moving forward in python.

File: trivia.py [hunk 2]
```diff
@@ -229,14 +230,14 @@ def _help(self, args, user, channel):
...
             self._cmsg(user, "Commands: score, standings, giveclue, help, "
                        "next, source")
             return
-        self._cmsg(user, "I'm nameless's trivia bot.")
+        self._cmsg(user, "I'm %s's trivia bot." % config.OWNER)
         self._cmsg(user, "Commands: score, standings, giveclue, help, next, "
                    "skip, source")
-        self._cmsg("Admin commands: die, set <user> <score>, start, stop, "
+        self._cmsg(user, "Admin commands: die, set <user> <score>, start, stop, "
```
Comment (rawsonj) [inline trivia.py]: Was this a bug here?

File: trivia.py (hunk 1 above)
Comment (andy5995) [inline trivia.py]: Yeah, np, Joe. I'll do that tomorrow.

**Comment by andy5995** [comment]
Not really a bug, but this would show up in the console - "_cmsg() takes exactly 3 arguments (2 given)"

**Pull request merged**